apps/communications/contacts/test/marionette/lib/contacts.js

```diff
@@ -386,10 +388,36 @@ Contacts.prototype = {
     }
   },
 
+  checkContactCreated: function() {
+    var ci = this.client.findElement(Contacts.Selectors.contactsIframe);
+    this.client.switchToFrame(ci);
+
+    var contactInContactList = this.client.findElement
+      (Contacts.Selectors.listContactFirst);
+    var contactName = contactInContactList.text();
+    return contactName;
+  },
+
   tapSave: function() {
     this.client.helper.waitForElement(Contacts.Selectors.formSave).click();
   },
 
+  tapSavefromDialer: function() {
+    this.client.helper.waitForElement(Contacts.Selectors.formGivenName);
+    this.client.helper.fillInputField(Contacts.Selectors.formGivenName, 'Name');
+    this.client.helper.waitForElement(Contacts.Selectors.formSave).click();
+    
+    var DialerOptionsView = require
```

Comment:
Ok, but then I do not find another way to do it than adding the funcions needed in dialer.js, which are the same than in dialer_options views.js...not sure then which would be the best solution...